mtags/src/main/scala/scala/meta/internal/metals/ControlCancellationException.scala

```diff
@@ -8,7 +8,7 @@ import scala.util.control.ControlThrowable
  */
 class ControlCancellationException(
     cause: CancellationException = new CancellationException()
-) extends CancellationException(cause.getMessage)
-    with ControlThrowable {
+) extends CancellationException(cause.getMessage) {
+  // with ControlThrowable { // TODO(gabro)
```

Comment:
I don't know enough about how this is used to understand how to fix it.

`ControlThrowable` has been made an abstract class in 2.13, so we can't mix it in this way anymore. See a relevant discussion at https://github.com/scala/bug/issues/11494